src/components/frame/Extensions/pages/ColonyDetailsPage/ColonyDetailsPage.tsx

```diff
@@ -1,59 +1,112 @@
 import React, { FC } from 'react';
 import clsx from 'clsx';
-import { FormattedMessage } from 'react-intl';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
 
-import styles from './ColonyDetailsPage.module.css';
 import { useColonyContext, useMobile } from '~hooks';
 import Avatar from '~v5/shared/Avatar';
-import Button, { TextButton } from '~v5/shared/Button';
+import Button from '~v5/shared/Button';
 import ObjectiveBox from '~v5/common/ObjectiveBox';
 import ExternalLink from '~shared/Extensions/ExternalLink';
 import { useSetPageHeadingTitle } from '~context/PageHeadingContext/hooks';
 import { formatText } from '~utils/intl';
 import { useActionSidebarContext } from '~context/ActionSidebarContext';
 import { ACTION } from '~constants/actions';
 import { ACTION_TYPE_FIELD_NAME } from '~v5/common/ActionSidebar/consts';
+import CopyableAddress from '~v5/shared/CopyableAddress';
+import Icon from '~shared/Icon';
+import { COLONY_LINK_CONFIG } from '~constants/colony';
+import { truncateText } from '~utils/string';
+import Tooltip from '~shared/Extensions/Tooltip';
+
+import styles from './ColonyDetailsPage.module.css';
 
 const displayName = 'frame.Extensions.pages.ColonyDetailsPage';
 
+const MAX_DESCRIPTION_LENGTH = 250;
+
+const MSG = defineMessages({
+  lockedToken: {
+    id: `${displayName}.locked-token`,
+    defaultMessage:
+      'This token is locked. Colony native tokens are locked and non-transferrable by default to avoid unwanted project token transfer outside of the colony.',
+  },
+});
```

Comment:
resolving this since we've decided this is the de facto way as per [this docs change](https://github.com/JoinColony/ColonyFrontEndLivingStandard/pull/22)